.github/workflows/ci_build_test.yml

```diff
@@ -132,7 +132,7 @@ jobs:
       - name: Rust Toolchain
         uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: nightly-2024-10-17
```

Comment:
Update the nightly toolchain to have access to the newly stabilized `feature(trait_upcasting)`